zeroweb-proto/proto/attachment.proto

```
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package io.github.xezzon.zeroweb.attachment;
```

Buf (1.59.0)

Files with package "io.github.xezzon.zeroweb.attachment" must be within a directory "io/github/xezzon/zeroweb/attachment" relative to root but were in directory "zeroweb-proto/proto".

(PACKAGE_DIRECTORY_MATCH)